numpy/lib/nanfunctions.py

```diff
@@ -508,7 +515,7 @@ def nansum(a, axis=None, dtype=None, out=None, keepdims=np._NoValue):
 
     Numpy integer arithmetic is modular. If the size of a sum exceeds the
     size of an integer accumulator, its value will wrap around and the
-    result will be incorrect. Specifying ``dtype=double`` can alleviate
+    result will be incorrect. Specifying ``dtype=np.double`` can alleviate
```

Comment:
After thinking about this more I think we should just remove the text because it is a secondary issue anyway, is covered by the `np.sum` documentation, and is bloating the documentation.  This is done in `3509317`.